blocks/api/registration.js

```diff
@@ -68,6 +68,12 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( 'focusable' in settings && ! ( isFunction( settings.focusable ) || isBoolean( settings.focusable ) ) ) {
```

Comment:
Sure!
Basically the problem is that the box around a block adds an additional element to the focus cycle that I wanted to remove but at the same time we need some way to focus blocks with no focusable components like the horizontal divider. Also with some blocks like the picture blocks they can be focused before you put the picture in as there is a button but after they have a picture there is nothing within the block to focus so the property needed to support a function.

Currently the property defaults to true so block owners will only have to care about it if their block has nothing focusable. I could also attempt to calculate post-render if there is anything focusable in a block and set the tabIndex of the surrounding div then.